docs/Acceptable-Casks.md

```diff
@@ -83,6 +83,17 @@ Casks which do not reach a minimum notability threshold (see [Rejected Casks](#r
 
 Note that none of these exceptions is a guarantee for inclusion, but examples of situations where we may take a second look.
 
+## Not a fork (usually)
+
+We will not add new casks using forks unless at least one of the following is true:
+
+* the fork has been "blessed" (i.e. pointed to by the original source repository and noted as the official successor)
+* the fork has been used as the replacement by at least two other major distributions (e.g. Ubuntu, Fedora, Arch Linux, not Some Random Linux Distro Nobody Uses)
```

Comment:
Nix, Alma Linux and Slackware won't like that

Reply:
```suggestion
* in at least two other major package distributors, the fork has replaced the original project.
  We consider Linux distributions Debian, Ubuntu, Fedora, Rocky, Alpine, Nix, Alma, Slackware, and Arch for this criterion.
```

Now With More Distros™

Reply:
~~But where's Gentoo?~~

It's hard to define which package distributors are "major" ones, and which are not. Listing all distros sounds a bit overkill, so I'd prefer the original language as it allows some flexibility.

Reply:
I think in then it would defer back to the original base to suggest using the fork as the replacement.

Reply:
Many Casks are proprietary software that don't necessarily have forks or alternative providers, though.

Reply:
This part of the docs is specifically talking about forks replacing the original base though.

Reply:
In context, it sounds like this applies to *all* forks submitted for inclusion, not just ones that aim to replace the original.

Reply:
👎🏻 on this change. I don't want to have a comprehensive list.

> I'm not sure this would work for a macOS-only cask.

It doesn't have to. I don't think there's much of an alternative in that case so we don't allow replacements due to forks.

> In context, it sounds like this applies to _all_ forks submitted for inclusion

This does apply, yes, so things should be named accordingly.

Reply:
Agreed with not needing a comprehensive list. Let's put down a couple of the obviously huge ones, throw an "etc." in there and call it good.

Reply:
```suggestion
* in at least two other major package distributors, the fork has replaced the original project.
  Consider foundational Linux distributions such as Debian, Fedora, and Arch for this criterion.
```

I feel like this is a compromise.

```suggestion
* the fork was chosen in place of the original project in at least two other major, foundational Linux distributions, e.g. Ubuntu, Fedora, and Arch.
```

This is closer to the original. I _really_ want to avoid the sentiment that "Some Random Linux Distro Nobody Uses" conveys. I think we can do that adequately with the word "foundational".

Reply:
Yeah I'm good with this. "Some Random Linux Distro Nobody Uses" is just some classic @MikeMcQuaid documenting 😂

Reply:
> I think we can do that adequately with the word "foundational".

I'm fine with this being reworded but it's pretty clear what "Some Random Linux Distro Nobody Uses" and I have no idea what "foundational" means.